itests/src/test/java/fi/metatavu/edelphi/test/mock/PanelMocker.java

```diff
@@ -0,0 +1,113 @@
+package fi.metatavu.edelphi.test.mock;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class PanelMocker extends AbstractMocker {
+  
+  private static final long PANELIST_ROLE_ID = 6l;
+  private static final long SYSTEM_USER_ID = 1l;
+  
+  private List<Long> panelIds = new ArrayList<>();
+  private List<Long> stampIds = new ArrayList<>();
+  
+  public PanelMocker mock() {
+    return this;
+  }
+  
+  public PanelMocker cleanup() {
+    for (Long stampId : stampIds) {
+      deleteStamp(stampId);
+    }
+    
+    for (Long panelId : panelIds) {
+      deletePanel(panelId);
+    }
+    
+    return this;
+  }
+
+  public void addCreatedPanel(String name) {
+    panelIds.add(findPanelByName(name));
+  }
+  
+  public long findPanelByRootFolder(Long rootFolderId) {
+    String sql = 
+      "SELECT " + 
+      "  id " + 
+      "FROM " + 
+      "  Panel " +  
+      "WHERE " + 
+      "  rootFolder_id = ?";
+    
+    return executeSqlLong(sql, rootFolderId);
+  }
+  
+  public long findPanelByName(String name) {
+    String sql = 
+      "SELECT " + 
+      "  id " + 
+      "FROM " + 
+      "  Panel " +  
+      "WHERE " + 
+      "  name = ?";
+    
+    return executeSqlLong(sql, name);
+  }
+
+  public long addPanelist(Long panelId, Long stampId, Long userId) {
+    long id = getNextId("PanelUser");
+    Date now = new Date();
+    addPanelUser(id, panelId, userId, PANELIST_ROLE_ID, false, SYSTEM_USER_ID, now, SYSTEM_USER_ID, now, "ADDED", stampId);
+    stampIds.add(id);
+    return id;
+  }
+    
+  protected void addPanelUser(Long id, Long panelId, Long userId, Long roleId, String joinType, Long stampId) {
+    Date now = new Date();
+    addPanelUser(id, panelId, userId, roleId, Boolean.FALSE, userId, now, userId, now, joinType, stampId);
+  }
+  
+  private void addPanelUser(Long id, Long panelId, Long userId, Long roleId, Boolean archived, Long creatorId, Date created, Long lastModifierId, Date lastModified, String joinType, Long stampId) {
+    String sql = 
+      "INSERT INTO " + 
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "INSERT INTO " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)